CODEOWNERS

```diff
@@ -2,7 +2,12 @@
 * @datalek @kin0992
 
 # Frontend codeowners
+apps/nextjs-website/public/icons/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/public/images/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/_contents/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/app/ @MarcoPonchia @marcobottaro @tommaso1
 apps/nextjs-website/src/components/ @MarcoPonchia @marcobottaro @tommaso1
 apps/nextjs-website/src/editorialComponents/ @MarcoPonchia @marcobottaro @tommaso1
-apps/nextjs-website/src/app/ @MarcoPonchia @marcobottaro @tommaso1
+apps/nextjs-website/src/lib/types/ @MarcoPonchia @marcobottaro @tommaso1
```

Comment:
Why not the whole `/lib/` folder?

Reply:
Because api.ts will deal with the CMS we want you to take part in the process